test/formula.jl

```diff
@@ -8,10 +8,10 @@
     # - support more transformations with I()?
 
     ## Formula parsing
-    import StatsModels: @formula, Formula, Terms
+    using StatsModels: @formula, Formula, Terms
 
     ## totally empty
-    t = Terms(Formula(nothing, 0))
+    t = Terms(@eval @formula $(:($nothing ~ 0)))
```

Comment:
This is intriguing...

Reply:
well since I've moved parsing into the `@formula` macro and there's not currently syntactic support for one-sided formulae this is what's necessary :)